recipes/rapidxml/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,38 @@
+#include <cstdio>
+#include <iostream>
+#include <fstream>
+#include <vector>
+
+#include "rapidxml.hpp"
+
+int main(void)
```

Comment:
This would become:
```
int main(int argc, char **argv)
{
  if (argc < 2) {
    std::cerr << "Need at least one argument\n";
    return 1;
  }
```